acp/acp_local.go

```diff
@@ -36,73 +81,94 @@ func (l *ACPLocal) Init(ctx context.Context, path string) {
 }
 
 func (l *ACPLocal) Start(ctx context.Context) error {
-	var localACP embedded.LocalACP
+	var manager runtime.RuntimeManager
 	var err error
 
 	if !l.pathToStore.HasValue() { // Use a non-persistent, i.e. in memory store.
-		localACP, err = embedded.NewLocalACP(
-			embedded.WithInMemStore(),
+		manager, err = runtime.NewRuntimeManager(
```

Comment:
suggestion: I think the code would be a bit more readable if the `if-else` just handles the construction/appending of `runtime.Opt`s, and the `runtime.NewRuntimeManager` call is moved outside of the `if-else` (and the err handling).

Reply:
good suggestion, thank you